Dockerfile

```diff
@@ -27,8 +27,14 @@ ENV NODE_ENV=production
 
 COPY package.json package-lock.json ./
 RUN apk add --no-cache --virtual .gyp python3 make g++ &&\
-    npm ci --only=production &&\
-    apk del .gyp &&\
+    npm ci --only=production
+
+# Install amqp library if RabbitMQ is enabled
+ARG ENABLE_RABBITMQ=false
+ENV ENABLE_RABBITMQ=${ENABLE_RABBITMQ}
+RUN if [ "$ENABLE_RABBITMQ" = "true" ]; then npm install amqplib@^0.10.5; fi
```

Comment:
Should we use a fixed version here?

Reply:
A fixed version avoids possibly breaking things without us noticing.

Suppose they release a v1.0.0 in the future with breaking changes. Our image could stop working without any intervention by us.

I think it's better than getting always the latest.
What do you think?

Reply:
Changed it after further alignment (outside of Github): https://github.com/HathorNetwork/hathor-wallet-headless/pull/522/commits/76ef4e8ac88d023ea8496bd866820f0e25de1f13